core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -59,7 +87,17 @@ oppia.factory('explorationData', [
             $log.info('Retrieved exploration data.');
             $log.info(response.data);
 
+            // Initialize the changeList by the one received from the backend.
             explorationData.data = response.data;
+            changeListService.loadAutosavedChangeList(
```

Comment:
@526avijitgupta: can you do these things in the function that calls getData(), and not here directly? That will remove the need for this service to depend on changeListService.